pinot-common/src/main/java/org/apache/pinot/common/messages/SegmentRefreshMessage.java

```diff
@@ -19,52 +19,60 @@
 package org.apache.pinot.common.messages;
 
 import java.util.UUID;
+import org.apache.helix.ZNRecord;
 import org.apache.helix.model.Message;
 
 
 /**
- * This (helix) message is sent from the controller to the server when a request is received to refresh
+ * This (Helix) message is sent from the controller to brokers and servers when a request is received to refresh
  * an existing segment.
  *
- * There is one mandatory field in the message -- the CRC of the new segment.
+ * NOTE: Changing this class to include new fields is a change in the protocol, so the new fields must be made optional,
+ * and coded in such a way that either controller, broker or server may be upgraded first.
```

Comment:
Can we update the javadoc to state the time boundary refresh is now done as part of SegmentRefreshMessage?

Reply:
Added